package.json

```diff
@@ -284,8 +284,9 @@
 		"ua-parser-js": "^0.7.28",
 		"underscore": "^1.13.1",
 		"underscore.string": "^3.3.5",
-		"url-polyfill": "^1.1.12",
-		"use-subscription": "^1.5.1",
+		"undici": "^4.2.2",
```

Comment:
bump to a newer version
https://www.npmjs.com/package/undici

Reply:
done